### PR TITLE
Add a simple debug script for k8s

### PR DESCRIPTION
**Pull request by supertetelman**
Addresses https://github.com/NVIDIA/deepops/issues/788

I am doing this in a shell script rather than a Ansible Playbook to make the log collection process more transparent for people who are less familiar with Ansible.

Basic tool to collect logs and basic debug information that solves 95% of the K8s problems users run into.

### CODE REVIEW

File: scripts/k8s/debug.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+kubectl get nodes > ${logdir}/get-nodes.log
+kubectl describe nodes > ${logdir}/describe-nodes.log
+for pod in $(kubectl get pods -n kube-system | grep nvidia-device-plugin | awk '{print $1}'); do
+  kubectl -n kube-system logs ${pod} > ${logdir}/get-plugin-logs-${pod}.log
+done
+
+# Helm
+helm list -aA > ${logdir}/helm-list.log
+
+# Packagin
```
Comment (michael-balint) [inline scripts/k8s/debug.sh:25]: Packagin --> Packaging.

**Comment by michael-balint** [comment]
This worked well for me and could be very useful when trying to determine the state of the k8s cluster.

@dholt - any thoughts/comments?

**Review by dholt — COMMENTED** [review]
Looks good. You might think about using a prefix on the logdir, like `log_*`. Also you might want to redirect stderr to the logs as well.

**Comment by supertetelman** [comment]
I made the requested documentations and also rebased the branch a bit so it doesn' pull in those other Docker config changes.

**Review by ajdecon — APPROVED** [review]
This looks great to me, and I think it will come in handy!

I'm approving because I don't want to block this, but here are two nice-to-haves that I'd like to see:

1. This script *implicitly* assumes it's running from the repository root. It would be good to explicitly ensure it runs correctly even if the current working dir isn't the repo root, by doing something like what [setup.sh does](https://github.com/NVIDIA/deepops/blob/master/scripts/setup.sh#L12).
1. This script assumes that the configuration directory is named `config` but DeepOps supports anything named `config*` in its gitignore. That's harder to detect automatically, but using a `CONFIG_DIR` [environment var with a default](https://github.com/NVIDIA/deepops/blob/e91a0feff99d152af5dd71cc8d72b185e6190ea6/scripts/setup.sh#L280) would also be nice.

That said, those aren't blockers! If you add these in, send me a ping and I'll approve again. If you don't have time right now, feel free to merge and I can add them in with a separate PR later. 😄

**Pull request merged**